ant-node/src/node.rs

```diff
@@ -160,11 +177,25 @@ impl NodeBuilder {
             None
         };
 
+        if !self.initial_peers.is_empty() && self.bootstrap_cache.is_some() {
```

Comment:
this block (180 - 191) of code can be down in : 
```rust
let initial_peers = match (self.initial_peers.is_empty(), self.bootstrap_cache) {
    (true, Some(_)) => return Err(Error::InitialPeersAndBootstrapCacheSet),
    (true, None) => self.initial_peers.clone(),
    (false, Some(cache)) => cache.get_sorted_addrs().cloned().collect(),
    (false, None) => vec![],
};
```
Which will be more readable ?

also, shall `initial_peers` always be preferred, i.e. it shall be `(true, _) => self.initial_peers.clone(),` ?

Reply:
Ahh.. we cannot get the initial bootstrap peers from the cache store anylonger as it won't be initialized anymore. 
The loading of cache happens at a higher layer and it is fed into `initial_peers`. I will modify according to this, thanks for the catch!